cmd/private-org-peribolos-sync/main.go

```diff
@@ -159,8 +159,19 @@ func generateRepositories(gc gitHubClient, orgRepos map[string]sets.String, logg
 
 // getReposForPrivateOrg itterates through the release repository directory and creates a map of
 // repository sets by organization that promote official images.
-func getReposForPrivateOrg(releaseRepoPath string) (map[string]sets.String, error) {
+func getReposForPrivateOrg(releaseRepoPath string, whitelist map[string][]string) (map[string]sets.String, error) {
 	ret := make(map[string]sets.String)
+
+	for org, repos := range whitelist {
```

Comment:
You might want to make `map[string]sets.String` into an exported type and expose an `Insert(org, repo string)` function for it to DRY this out, so can share code.

Reply:
Every tool is using its maps in a different manner, so there is no reason to expose the logic. Also, we don't need to insert anything in the existing whitelist.